### PR TITLE
Making SAML entityId configurable (#187)

### PR DESCRIPTION
**Pull request by ssciolla**
This PR makes a minor modification to `settings.py` to allow for users to specify a `SAML2_ENTITYID` as an environment variable,. If unspecified, the default we fall back upon is still to construct the `entityId` based on the base URL + `metadata/`. This change will allow us to move in the future toward global `entityId`s that can be shared among other T&L applications. I also commented out and moved a variable that I previously didn't realize was part of the now-unused cron. This PR aims to resolve issue #187.

Note: these changes are currently deployed to `test-studentexplorer.tl.it.umich.edu` for testing purposes.

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -21,9 +21,6 @@ DJANGO_DEBUG=true
 # Debug level can conflict with Django Debug Toolbar
 DJANGO_LOGGING_LEVEL=INFO
 
-# department affiliation string from MCommunity settings
```
Comment (zqian) [inline .env.sample]: Don't think this file diff is needed or is relevant to the issue.

Comment (jonespm) [reply .env.sample]: Yeah, looks like just a re-org, not related but a cleanup.

File: student_explorer/settings.py [hunk 2]
```diff
@@ -190,10 +190,11 @@
     SAML2_REMOTE_PEM_FILE = config('STUDENT_EXPLORER_SAML_REMOTE_PEM_FILE',default='')
 
     SAML2_DEFAULT_IDP = config('STUDENT_EXPLORER_SAML_DEFAULT_IDP',default='')
-
     if SAML2_DEFAULT_IDP:
         SAML2_DEFAULT_IDP = '?idp=%s' % SAML2_DEFAULT_IDP
 
+    SAML2_ENTITYID = config('SAML2_ENTITYID', default=SAML2_URL_BASE + 'metadata/')
```
Comment (jonespm) [inline student_explorer/settings.py]: Looks like we don't mention any of these other SAML values in the sample. We probably should document them somewhere but that could be on another ticket.

File: .env.sample (hunk 1 above)
Comment (ssciolla) [reply .env.sample]: It's not relevant, but it's something that needed to happen (either be commented out or removed). I can do it in another issue if you would like.

File: student_explorer/settings.py (hunk 2 above)
Comment (ssciolla) [reply student_explorer/settings.py]: Yeah, I was wondering about that. I thought maybe it had something to do with how we configure environment variables through OpenShift (i.e. how we aren't set up to use a Config Map). I think we can create another issue that addresses this (and maybe what I tried to sneak in but @zqian called me out on).

File: .env.sample (hunk 1 above)
Comment (ssciolla) [reply .env.sample]: Okay, @zqian, I've relented, see my latest commit undoing the change. I will create a separate issue about this (and maybe what @jonespm mentioned below).

**Pull request merged**